src/interviewer.t

```diff
@@ -217,17 +235,70 @@ warner_interviewer : Person 'interviewer' 'Harried Interviewer' @interview_room
 ;
 
 ++ SpecialTopic 'anime club' ['anime', 'anime club', 'japanese', 'japanese club', 'weeaboo', 'weeb']
-    ""
+    ""//"<q>私はアニメが大好きです~~~!</q>"
```

Comment:
latest commit includes another build flag (usage: `. build -q`) for usage with QTads, which is a standalone application that also supports full unicode. (idk if it works on anything but OS X, it's just the following: `open -a QTads tgt/cmu_adv_cmd.t3`)